Packs/TrendMicroVisionOne/IncidentFields/incidentfield-Trend_Micro_Vision_One_XDR_Updated_Time.json

```diff
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Trend Micro Vision One XDR Incident"
+    ],
+    "caseInsensitive": true,
+    "cliName": "trendmicrovisiononexdrupdatedtime",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_trendmicrovisiononexdrupdatedtime",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Trend Micro Vision One XDR Updated Time",
```

Comment:
@mikedgibson What is the value of this field represents? Is it the updated time for a detection/alert?
If so, I believe the existing field "Detection Update Time" can cover for it.